lib/liquid/c.rb

```diff
@@ -59,30 +59,33 @@ def self.new(source, line_numbers = false, line_number: nil, for_liquid_tag: fal
 
 Liquid::ParseContext.class_eval do
   class << self
-    attr_accessor :disable_liquid_c_nodes
+    attr_accessor :disabled_liquid_c_nodes
   end
-  self.disable_liquid_c_nodes = false
+  self.disabled_liquid_c_nodes = false
 
   alias_method :ruby_new_block_body, :new_block_body
 
   def new_block_body
-    if disable_liquid_c_nodes
+    if disabled_liquid_c_nodes
       ruby_new_block_body
     else
       Liquid::C::BlockBody.new
     end
   end
 
   # @api private
-  def disable_liquid_c_nodes
+  def disabled_liquid_c_nodes
```

Comment:
IMO `disabled_liquid_c_nodes` sounds like it would contain a list of liquid-c nodes that are disabled. How about `liquid_c_nodes_disabled` and then we can do:

```suggestion
  def liquid_c_nodes_disabled?
```